npm/pod.go

```diff
@@ -126,21 +126,21 @@ func (npMgr *NetworkPolicyManager) UpdatePod(oldPodObj, newPodObj *corev1.Pod) e
 		return nil
 	}
 
+	if isInvalidPodUpdate(oldPodObj, newPodObj) {
```

Comment:
Isnt this already checked in?

Reply:
Yes it did get checked in but this might be stale code from all the branch changes we did. since this was a single commit, I could not cut it down